content/en/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definition-versioning.md

```diff
@@ -961,5 +961,12 @@ The following is an example procedure to upgrade from `v1beta1` to `v1`.
     the same content. This forces the backend to write objects in the current
     storage version, which is `v1`.
 2. Remove `v1beta1` from the CustomResourceDefinition `status.storedVersions` field.
+This can be done by accessing kubernetes api with patch request
+```bash
```

Comment:
How about adding a comment that readers should set `$resource_name` to a suitable value first? (I can imagine seeing an issue filed in the future by someone who has missed a detail).

(The Kubernetes docs convention is to [use angle brackets for placeholders](https://kubernetes.io/docs/contribute/style/style-guide/#use-angle-brackets-for-placeholders) but the Bourne shell already has a different idea about those mean).